Framework/HistogramData/src/HistogramRebin.cpp

```diff
@@ -132,30 +132,30 @@ Histogram rebinFrequencies(const Histogram &input, const BinEdges &binEdges) {
       } else {
         factor = 1 / nwidth;
         ynew[inew] *= factor;
-        enew[inew] = sqrt(enew[inew]) * factor;
+        enew[inew] = sqrt(enew[inew] * factor);
```

Comment:
@SimonHeybrock  I made this change and seem to be getting the correct values for frequencyStandardDeviations.